doc/source/cluster/guide.rst

```diff
@@ -0,0 +1,275 @@
+.. _deployment-guide:
+
+Ray Deployment Guide
+====================
+
+This page provides an overview of how to deploy a multi-node Ray cluster, including how to:
+
+* Launch the cluster.
+* Set up the autoscaler.
+* Monitor a multi-node cluster.
+* Best practices for setting up a Ray cluster.
+
+Launching a Ray cluster
+-----------------------
+
+There 2 recommended ways of launching a Ray cluster are via:
+
+1. :ref:`The cluster launcher <cluster-cloud>`
+2. :ref:`The kubernetes operator <Ray-operator>`
+
+Cluster Launcher
+^^^^^^^^^^^^^^^^
+
+The goal of :ref:`the cluster launcher <cluster-cloud>` is to make it easy to deploy a Ray cluster on
+any cloud. It will:
+
+* provision a new instance/machine using the cloud provider's SDK.
+* execute shell commands to set up Ray with the provided options.
+* (optionally) run any custom, user defined setup commands.
+* Initialize the Ray cluster.
+* Deploy an autoscaler process.
+
+Kubernetes Operator
+^^^^^^^^^^^^^^^^^^^
+
+The :ref:`k8s operator <Ray-operator>` serves a very similar purpose to the
+cluster launcher, but follows the `kubernetes operator pattern
+<https://kubernetes.io/docs/concepts/extend-kubernetes/operator>`__. It's
+defined as :ref:`a helm chart <Ray-helm>` and will:
+
+* Create a controller (the operator).
+* Provision a new pod (head node).
+* execute shell commands to set up Ray with the provided options.
+* (optionally) run any custom, user defined setup commands.
+* Initialize the Ray cluster.
+
+The operator will then serve as the autoscaler.
```

Comment:
@DmitriGekhtman to review

Reply:
I'll follow up with touch-ups to k8s info in a separate pr.